server/config/validation/edit-juror-details-mod.js

```diff
@@ -11,13 +11,73 @@
     messageMap = {
       primaryPhone: 'Telephone number cannot contain letters or special characters apart from hyphens, dashes, brackets or a plus sign.',
       secondaryPhone: 'Telephone number cannot contain letters or special characters apart from hyphens, dashes, brackets or a plus sign.',
+      'thirdParty-mainPhone': 'Telephone number cannot contain letters or special characters apart from hyphens, dashes, brackets or a plus sign.',
+      'thirdParty-secPhone': 'Telephone number cannot contain letters or special characters apart from hyphens, dashes, brackets or a plus sign.',
```

Comment:
I think we can make these a single item since they're all the same message

Reply:
This is required as we pass this to another function that checks the map for the input name to get the error message